fixtures/cameo/root-par-6.json

```
@@ -0,0 +1,753 @@
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [62, 69],
+          "type": "ColorPreset",
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [70, 77],
+          "type": "ColorPreset",
+          "comment": "Lavender"
+        },
+        {
+          "dmxRange": [78, 85],
+          "type": "ColorPreset",
+          "comment": "Mauve"
+        },
+        {
+          "dmxRange": [86, 93],
+          "type": "ColorPreset",
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "Warm White"
+        },
+        {
+          "dmxRange": [110, 117],
+          "type": "ColorPreset",
+          "comment": "White"
+        },
+        {
+          "dmxRange": [118, 125],
+          "type": "ColorPreset",
+          "comment": "Cold White"
+        },
+        {
+          "dmxRange": [126, 201],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [202, 207],
+          "type": "ColorPreset",
+          "comment": "User Color_1"
+        },
+        {
+          "dmxRange": [208, 213],
+          "type": "ColorPreset",
+          "comment": "User Color_2"
+        },
+        {
+          "dmxRange": [214, 219],
+          "type": "ColorPreset",
+          "comment": "User Color_3"
+        },
+        {
+          "dmxRange": [220, 225],
+          "type": "ColorPreset",
+          "comment": "User Color_4"
+        },
+        {
+          "dmxRange": [226, 255],
+          "type": "ColorPreset",
+          "comment": "UV"
+        }
+      ]
+    },
+    "Amber": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe 2": {
+      "name": "Strobe",
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "20Hz"
+        }
+      ]
+    },
+    "Dimmer 4": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Dimmer fein": {
+      "capability": {
+        "type": "Intensity"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "3-Channel",
+      "shortName": "3ch",
+      "channels": [
+        "Dimmer",
+        "Color Presets",
+        "DMX Delay"
+      ]
+    },
+    {
+      "name": "5-Channel #1",
+      "shortName": "5ch#1",
+      "channels": [
+        "Dimmer 2",
+        "Strobe Multifuntional",
+        "Color Presets",
+        "Sound Sensitivity",
+        "DMX Delay"
```

@FloEdelmann Not sure if there are guidelines for Channel Mode enumeration?

All Changes are derived from previous renamings.
```suggestion
      "name": "5-Channel #1",
      "shortName": "5ch#1",
      "channels": [
        "Dimmer",
        "Strobe Multifuntional",
        "Color Preset",
        "Sound",
        "DMX Delay"
```

---

There are some unwritten rules, based on consistency with existing fixtures: E.g. this should be `5-channel 1` / `5ch1`; if there was only one mode with 5 channels, it should be `5-channel` / `5ch`.